metricbeat/module/kubernetes/container/container_test.go

```diff
@@ -21,47 +21,78 @@
 package container
 
 import (
-	"io/ioutil"
+	"io"
 	"os"
 	"testing"
-	"time"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/elastic/beats/v7/metricbeat/module/kubernetes/util"
 )
 
 const testFile = "../_meta/test/stats_summary.json"
 
-func TestEventMapping(t *testing.T) {
+type ContainerTestSuite struct {
+	suite.Suite
+	MetricsRepo          *util.MetricsRepo
+	NodeName             string
+	Namespace            string
+	PodName              string
+	ContainerName        string
+	AnotherContainerName string
```

Comment:
why do we need `AnotherContainerName` if this is a test for the container?
as I understand it is not used, no?